graphql/resolve/query_rewriter.go

```diff
@@ -1211,7 +1212,9 @@ func buildAggregateFields(
 				}
 				aggregateChild := &gql.GraphQuery{
 					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
-					Attr:  strings.ToLower(function) + "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
+					Attr: strings.ToLower(
+						function,
```

Comment:
This still bothers me :).... how about

```golang
				aggregateChild := &gql.GraphQuery{
					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
					Attr: strings.ToLower(function) + 
                                             "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
				}

```

Reply:
I will fix it up tomorrow, current plan is to get this to a good shape as soon as we have a minimal test-harness running (which we have atm ~40% coverage, and if we open up a few more folders - we will be in a better shape to accept changes more confidently ... particularly for `GraphQL` folder.

Reply:
@matthewmcneely thanks for providing the new structure .. I have manually fixed it here https://github.com/dgraph-io/dgraph/pull/8191/commits/e378950dc958730782f23a61f1dbe19a3a3c7a8a